cpp/data/rss_email.template.en

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+    <body>
+Dear ${user_name},
+
+The following items from your selected RSS feeds are new:</br>
+    <table id="table_delivered">
+        <thead>
+            <tr>
+                <th>Title</th>
+                <th>Description</th>
```

Comment:
Descriptions may contain lots of html tags which might screw up the table layout. I'd rather recommend to remove the descriptions column entirely and replace it with the name of the RSS feed instead (maybe including a hyperlink to it).